.github/CONTRIBUTING.md

```diff
@@ -3,49 +3,60 @@
 :wave: Hi there!
 We're thrilled that you'd like to contribute to this project. Your help is essential for keeping it great.
 
-## Submitting a pull request
+## How to Contribute
 
-[Pull Requests][pulls] are used for adding new playbooks, roles, and documents to the repository, or editing the existing ones.
+### 1. Create an issue
 
-### Pre-requisites
+Report problems or suggest improvements by [creating an issue](https://github.com/oxsecurity/megalinter/issues).
 
-- You need [**Python 3**](https://www.python.org/downloads/) (version 3.7 minimum) and [**Node.js** (14+)](https://nodejs.org/en/download/) to be installed on your computer.
-  - If you have issues running Python on Windows, you can uninstall it then reinstall it using [this video tutorial](https://www.youtube.com/watch?v=uDbDIhR76H4), then copy python.exe and name it python3.exe
+### 2. Fork the project
 
-- Run the following commands at the root of the repository to install required dev dependencies
-```shell
-python3 -m venv .venv
-. .venv/bin/activate
-echo ".venv/" >> .git/info/exclude
-python3 -m pip install -U pip
-python3 -m pip install -r requirements.dev.txt
-```
+[Fork the repository](https://github.com/oxsecurity/megalinter) to your GitHub account.
 
-_If it doesn't work, just run the following script_
+### 3. Make changes
 
-```shell
-pip install -r requirements.dev.txt
-pip install mkdocs-material "mkdocs-glightbox==0.3.2" "pymdown-extensions==9.11"
-npm install markdown-table-formatter -g
-```
+Clone your fork locally and make the necessary changes:
 
-Second level dev dependencies are installed by running `./build.sh` which is also a test if the installation worked
-
-```shell
-./build.sh
-2021-03-30 19:40:03,790 [INFO] Validating ansible.megalinter-descriptor.yml
-2021-03-30 19:40:03,879 [INFO] Validating arm.megalinter-descriptor.yml
-…
-Formatting markdown tables…
-Need to install the following packages:
-  markdown-table-formatter
-Ok to proceed? (y)
-…
-INFO    -  Documentation built in 9.76 seconds
-(done.)
+```bash
+git clone git@github.com:YOURNAMESPACE/megalinter.git
 ```
 
-_(if you have a permission denied issue on Windows, please check [this solution](https://stackoverflow.com/a/57168165/7113625))_
+### 4. Test your changes
+
+#### 4.1 Gitpod
+
+Use Gitpod for a cloud-based development environment:
+
+1. Sign up for Gitpod: <https://gitpod.io>
+2. Fork the `megalinter` repository
+3. Open your fork in Gitpod: `https://gitpod.io/#https://github.com/username/megalinter`
+4. Create a new branch: `git checkout -b my-feature-branch`
+5. Make your changes and commit: `git add .` and `git commit -m "chore: description of changes"`
+6. Test all : `make tests` or `make tests-fast` for TDD mode
+7. Test with megalinter: `make megalinter-tests`
+8. Push your changes: `git push origin my-feature-branch`
+9. Create a pull request on GitHub
+10. Wait for a review
+
+Keep your Gitpod workspace synced with the main repository.
```

Comment:
For trying out how Gitpod works for a repo you don't own, I think we can reformulate a little by not requiring that you sign up for Gitpod, but more like what we see in https://www.gitpod.io/docs/introduction/getting-started#step-1-your-first-workspace


> ## Step 1: Your first workspace
> 
> `youtube: ij1msCffQZA`
> 
> The best way to get see the power of Gitpod, is to try it out by starting your first [Workspace](/docs/configure/workspaces).
> 
> To start your first workspace:
> 
> 1. Navigate to a GitHub, GitLab or Bitbucket repository.
> 2. Open the repo in Gitpod by prefixing the URL with: `gitpod.io/#`.
> 3. Confirm the workspace creation.
> 
> For example, click this link to open the NodeJS project in a Gitpod workspace:
> 
> [`https://gitpod.io/#https://github.com/nodejs/node`](https://gitpod.io/#https://github.com/nodejs/node)
> 

And adding that you might need to press "Continue with GitHub" to log in, and authorize Gitpod (the permissions) it if it is the first time you use Gitpod.
![image](https://github.com/oxsecurity/megalinter/assets/27212526/40379827-b07e-43dc-8ef6-e29668a63b95)
![image](https://github.com/oxsecurity/megalinter/assets/27212526/b8430174-bc3e-4b0e-9390-eb144e5d8709)
and the last authorization page I can't see since I'm not new.